spec/controllers/releases_controller_spec.js

```diff
@@ -199,6 +201,8 @@ describe("controller: ReleasesController", function() {
     it("should be possible to open the edit modal", function() {
       this.$httpBackend.expectGET('/api/releases')
       .respond(200, JSON.stringify(sample_releases));
+      this.$httpBackend.expectGET('/api/releases/columns/product')
+      .respond(200, JSON.stringify({product: ['Product1', 'Product2'], count: 2}));
       this.$httpBackend.flush();
```

Comment:
The existence of these two lines raise a "Unsatisfied request" error while their absence raise a "Unexpected request" error..